src/CloudinaryAdapter.php

```diff
@@ -35,15 +40,15 @@ public function __construct(string $config)
      *
      * @param string $path
      * @param string $contents
-     * @param Config $options   Config object
+     * @param Config $options Config object
      *
      * @return array|false false on failure file meta data on success
      */
     public function write($path, $contents, Config $options)
     {
         fwrite(tmpfile(), $contents);
 
-        return $this->writeStream($path, $tempFile, $options);
+        return $this->writeStream($path, $tempFile, $options); // [CN] Where is $tempFile defined?
```

Comment:
$should be $tempFile = tmpfile();

I renamed it. I'll just add that back @const-cloudinary